news/2021-11-08-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,132 @@
+
+This change introduces an algorithmic assessment of the complexity of note patterns in historical context and will introduce a more accurate assessment for maps that feature atypical combinations of triples, doubles, quints and much more!
+
+Consider these two rhythms:
+
+![](/wiki/shared/news/2021-11-08-performance-points-star-rating-updates/rhythm-complexity.png)
+
+Before this difficulty calculation change, the first pattern would have always had greater strain than the second.  In reality, the second can be as demanding or even harder (or potentially harder) than the first depending on the map. With these new changes, the algorithm now more accurately reflects this.
+
+These changes are exceedingly complex from a mathematical point of view. To keep things as simple as possible, notes are grouped up into 'islands', with a triple expressed as a size 3 'island'. Each island is then grouped by the algorithm and considered in a variety of ways, such as:
+
+- islands becoming smaller or larger (size 5 to size 3, for example)
+- islands which change from even to odd sizes
+- the ratio of time elapsed between islands (changes like 1/4 streams into 1/3 streams)
+
+These changes all collectively combine to produce a rhythm value which then adjusts the difficulty of a pattern, from which star rating is ultimately derived.
+
+We hope this change will bring a renewed interest in rhythmically complex songs on osu! and will incentivize players to improve their finger control. 
+
+Good luck setting new, impressive top plays that showcase your skills!
+
+### Total star rating adjustments
+
+In a nutshell, osu! difficulty calculation processes a map and produces skill rating values. These skill values are then manipulated in two different ways to produce two different numbers that you’ll be familiar with: star rating and performance points. 
+
+The correlation between these two numbers has now been improved, following the mathematical derivation work from [**Naitsirk**](https://osu.ppy.sh/users/8202998) in [this pull request](https://github.com/ppy/osu/pull/13986). 
+
+To summarize these changes briefly, maps which focus on one skill will now receive lower star rating than ones which focus on both. **These changes do not affect performance points.**
+
+Here are a few examples:
+
+- [DragonForce - Through the Fire and Flames (Ponoyoshi) [Myth]](https://osu.ppy.sh/beatmapsets/382400#osu/1001682) 6.1* -> 6.32*
+- [Panda Eyes & Teminite - Immortal Flame (feat. Anna Yvette) (Sotarks) [Eternal Blaze]](https://osu.ppy.sh/beatmapsets/703957#osu/1489207) 7.05* -> 6.94*
+
+As for the formula change itself, you can compare and contrast between these:
+
+Old formula: ![](/wiki/shared/news/2021-11-08-performance-points-star-rating-updates/old-sr-white.png)
+New formula: ![](/wiki/shared/news/2021-11-08-performance-points-star-rating-updates/new-sr-white.png)
```

Comment:
if the images are to be kept in the article they should be figures instead of inline. "\_ formula" can be the captions